openttd/Dockerfile

```diff
@@ -1,19 +1,31 @@
 FROM base
 
-RUN apt-get install -y libsdl1.2debian
-RUN apt-get install -y libfontconfig1
+RUN apt-get update && \
+	apt-get install -y libsdl1.2debian \
+	libfontconfig1 && \
+	adduser openttd && \
+	mkdir -p /home/openttd/extract && \
+	chown -R openttd:openttd /home/openttd/extract
 
-RUN adduser openttd && mkdir -p /openttd/extract && chown -R openttd:openttd /openttd
-WORKDIR /openttd
+USER openttd
+WORKDIR /home/openttd
```

Comment:
For consistency with every other folder in this repo, please leave it at `/openttd`, not in `/home`.